src/lib/mina_lib/mina_lib.ml

```diff
@@ -1062,8 +1062,8 @@ let last_epoch_delegators t ~pk =
   in
   find_delegators last_epoch_delegatee_table pk
 
-let perform_compaction t =
-  match Mina_compile_config.compaction_interval_ms with
+let perform_compaction compaction_interval_ms t =
```

Comment:
Consider changing parameter to ~(compaction_interval : Time.Span.t) (and performing conversion somewhere on top level)